localstack/services/ses/provider.py

```diff
@@ -196,31 +217,36 @@ def get_identity_verification_attributes(
             VerificationAttributes=attributes,
         )
 
-    @handler("SendEmail")
+    @handler("SendEmail", expand=False)
```

Comment:
I think the handler is "expanded" in the pro provider, which leads to the test failing with the pro integration because of a `super()` call. Sorry for not picking that up earlier. You might need to use the expanded version.